integrations/slack/src/misc/utils.ts

```diff
@@ -337,3 +337,31 @@ export const validateRequestSignature = ({ req, logger }: { req: Request; logger
     return false
   }
 }
+
+const updateBotpressBotName = async (client: Client, ctx: IntegrationCtx) => {
+  const { botName } = ctx.configuration
+
+  if (botName?.trim()) {
```

Comment:
**nitpick:** declare variable `const trimmedName = botName?.trim()` and reuse below

Reply:
The PR has been rebased and this is now fixed